dll/win32/iphlpapi/icmp.c

```diff
@@ -1,1129 +1,575 @@
 /*
- * ICMP
- *
- * Francois Gouget, 1999, based on the work of
- *   RW Hall, 1999, based on public domain code PING.C by Mike Muus (1983)
- *   and later works (c) 1989 Regents of Univ. of California - see copyright
- *   notice at end of source-code.
- * Copyright 2015 Sebastian Lackner
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ * PROJECT:         ReactOS IP Helper API
+ * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
+ * FILE:            dll/win32/iphlpapi/icmp.c
+ * PURPOSE:         ICMP functions
+ * PROGRAMMERS:     Tim Crawford <crawfxrd@gmail.com>
```

Comment:
Was this already reviewed in another PR or should it be reviewed again here?
Also, you may want to create a commit that credits Tim as author.

Reply:
It was never reviewed. I think I was trying to get [ROSTESTS-214](https://jira.reactos.org/browse/ROSTESTS-214) in first, since those tests were the basis for my implementation (so we could see the results in testman before/after and in Windows 2003 vs ReactOS).

Reply:
I will look into the tests too